### ISSUE TITLE
How to know whether incoming or outgoing call in `didReceiveStartCallAction` event

### ISSUE THREAD

**Issue by sujameslin** [ianlin/react-native-callkit#28]
When user start call action from Recents in built-in Phone app, how to know that if clicked call history item was incoming or outgoing call?
Is it sending some parameter when `didReceiveStartCallAction` is invoked?

**Comment by aarkalyk** [comment]
I don't think we can get information about wether call was incoming or outgoing.
The `RNCallKitHandleStartCallNotification` is sent from the following method
```
+ (BOOL)application:(UIApplication *)application
continueUserActivity:(NSUserActivity *)userActivity
  restorationHandler:(void(^)(NSArray * __nullable restorableObjects))restorationHandler
```
Just checked if `userActivity` provides such information, turns out it doesn't. 
Why do you need it anyway?
